src/credit-card/card-number/utils/card-types.ts

```diff
@@ -16,7 +16,7 @@ export const cardTypeConsts = [
   {
     name: 'American Express',
     lengths: [15],
-    prefixExpression: '3[4,7]',
+    prefixExpression: '3[47]',
```

Comment:
Ya that'll match a comma there... I strip non digits elsewhere so that bug won't be encountered but that regex is wrong with the comma in it.